dpppt-config-backend/src/main/resources/i18n/messages_en.properties

```diff
@@ -617,12 +617,12 @@ en:
     checkin_ended_text_detailed: The check-in function is automatically deactivated after a positive test result. It can be reactivated after the end of isolation.
     checkins_create_qr_code_subtitle: Enter a title that describes the location or your event in more detail. This title will be displayed on check-in.
     checkin_report_heading: What can you do?
-    checkin_report_title1: Protect your contacts
+    checkin_report_title1: \nProtect your contacts
```

Comment:
Is the newline character here intentional? None of the other translations appear to have one.